doc/api/resources/reusablemedia.rst

```diff
@@ -21,12 +21,15 @@ id                                    integer                    Internal ID of
 type                                  string                     Type of medium, e.g. ``"barcode"``, ``"nfc_uid"`` or ``"nfc_mf0aes"``.
 organizer                             string                     Organizer slug of the organizer who "owns" this medium.
 identifier                            string                     Unique identifier of the medium. The format depends on the ``type``.
+secret                                string                     Secret to identify the medium (or ``null``)
```

Comment:
That appears confusing. When comparing this with OrderPosition or GiftCard together with this description, one could think that `secret` is the thing that is printed on the medium and scanned, such as the barcode content. However, that is not what this `secret` is for. If I recall correctly, this is more like a "token to add this to a customer account". 

At the minimum, the description here should make that clearer, but ideally we find another name than "secret" that makes the purpose more clear. Could very well be that we need other kinds of secret things on this model in the future. Maybe `claim_token` or `claim_secret`?